docs/pages/smart-contracts/deployed-addresses.mdx

```diff
@@ -70,21 +70,21 @@ The following tables list the deployed factory and account implementation contra
 | Base             | `0x0000000000006E2f9d80CaEc0Da6500f005EB25A` |
 | Base Sepolia     | `0x0000000000006E2f9d80CaEc0Da6500f005EB25A` |
 
-## `Modular Account V2 Modules` (v2.0.0)
+## `Modular Account V2 Modules` (v2.0.0 except for AllowlistModule)
```

Comment:
Can we omit this? It's likely more confusing than helpful, to new readers, and the version callout is present in the name for allowlist.